packages/loader/src/gltf/parser/GLTFAnimatorControllerParser.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AnimationClip,
+  AnimatorController,
+  AnimatorControllerLayer,
+  AnimatorStateMachine
+} from "@galacean/engine-core";
+import { GLTFParser } from "./GLTFParser";
+import { GLTFParserContext, GLTFParserType, registerGLTFParser } from "./GLTFParserContext";
+
+@registerGLTFParser(GLTFParserType.AnimatorController)
+export class GLTFAnimatorControllerParser extends GLTFParser {
+  parse(context: GLTFParserContext): Promise<AnimatorController> {
+    return context.get<AnimationClip>(GLTFParserType.Animation).then((animations) => {
+      if (context.needAnimatorController) {
+        const animatorController = this._createAnimatorController(animations);
+        return Promise.resolve(animatorController);
+      } else {
+        return Promise.resolve(null);
```

Comment:
If `needAnimatorController` is false, why should wait animations loaded(even if animations is empty)

Reply:
done